src/config/contracts.ts

```diff
@@ -1,17 +1,21 @@
-export const Addresses = {
+import { config } from 'src/config/config';
+
+const ALFAJORES_ADDRESSES = {
+  Accounts: '0xed7f51A34B4e71fbE69B3091FcF879cD14bD73A9',
+  Election: '0x1c3eDf937CFc2F6F51784D20DEB1af1F9a8655fA',
+  EpochManager: '0x106d8A1192fB0E58079487d093b42332e66F5d8f',
+  Governance: '0xAA963FC97281d9632d96700aB62A4D1340F9a28a',
+  LockedGold: '0x6a4CC5693DC5BFA3799C699F3B941bA2Cb00c341',
+  Validators: '0x9acF2A99914E083aD0d610672E93d14b0736BBCc',
+} as const;
+
+const MAINNET_ADDRESSES = {
   Accounts: '0x7d21685C17607338b313a7174bAb6620baD0aaB7',
-  DoubleSigningSlasher: '0x50C100baCDe7E2b546371EB0Be1eACcf0A6772ec',
-  DowntimeSlasher: '0x71CAc3B31c138F3327C6cA14f9a1c8d752463fDd',
   Election: '0x8D6677192144292870907E3Fa8A5527fE55A7ff6',
-  EpochRewards: '0x07F007d389883622Ef8D4d347b3f78007f28d8b7',
-  GasPriceMinimum: '0xDfca3a8d7699D8bAfe656823AD60C17cb8270ECC',
-  GoldToken: '0x471EcE3750Da237f93B8E339c536989b8978a438',
+  EpochManager: '0x0000000000000000000000000000000000000000', // to be added
```

Comment:
thoughts on switching it to use the registery for this contract at least so we can merge and not have to remember to come back here right before L2 launch